src/App.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './App.css'
+import play from './access/paly.svg'
+import pause from './access/pause.svg'
+import resetTime from './access/time.svg'
+import arrow from './access/arrow.svg'
+import {connect} from "react-redux";
+import {addBreakLength, addSessionLength, reset, startStopTimer} from "./redux/appReduxer";
+import {getFormate} from "./utils";
```

Comment:
Use white spaces between curly brackets, it helps to make import statements more readable

```ts
import { someModule } from "some-module" // ✅
import {someModule} from "some-module" // ❌
```

Reply:
`import { someModule } from "some-module"`
Set up IDE - now everything will be fine